src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -5289,6 +5305,33 @@
       commands:
           BasicInformation:
               - MfgSpecificPing
+      enum values:
+          EthernetNetworkDiagnostics:
+              # Both the enum and the values got renamed, so don't output the
+              # new values with the old enum name or the old values with the
+              # new enum name.
+              PHYRateType:
```

Comment:
I do not know what this file is used for, but I'm curious why the old enum and value names being added here?

Reply:
This file is used for listing, for each release the Darwin Matter.framework, which things got added, deprecated, removed, renamed, etc in that release.  That then controls the codegen output for the generated APIs.

The specific section you are looking at there is the "removed" section, and it's listing the old names with the new enum name and the new names with the old enum name, to prevent codegen of those values at all.